src/lib.rs

```diff
@@ -44,16 +64,15 @@ fn default_build_command() -> process::Command {
 }
 
 impl Build {
-    pub fn execute(
-        self,
-        crate_name: &str,
-        static_dir_path: impl AsRef<Path>,
-        build_dir_path: impl AsRef<Path>,
-    ) -> Result<()> {
-        log::trace!("Build: get package's metadata");
+    pub fn execute(self, crate_name: &str, static_dir_path: impl AsRef<Path>) -> Result<PathBuf> {
```

Comment:
It would probably make the API nicer to move static_dir_path to a field too (use builder pattern)